Source/Charts/Data/Implementations/Standard/BarChartDataEntry.swift

```diff
@@ -62,32 +66,27 @@ open class BarChartDataEntry: ChartDataEntry
         calcPosNegSum()
         calcRanges()
     }
-        
+
     /// Constructor for stacked bar entries. One data object for whole stack
-    @objc public init(x: Double, yValues: [Double], data: AnyObject?)
+    @objc public convenience init(x: Double, yValues: [Double], icon: NSUIImage?)
```

Comment:
could you double check here, the diff tool seems weird that the old function is `@objc public init(x: Double, yValues: [Double], data: AnyObject?)` and the new one is `@objc public convenience init(x: Double, yValues: [Double], icon: NSUIImage?)`

it's harder to say whether this is a expected one.

Reply:
I changed the order of the init that takes data and the one that takes an icon to make all of the files declare those initializers in the same order.